docs/src/integrate/nodejs-api.md

```diff
@@ -834,6 +837,10 @@ const Linter = require("eslint").Linter;
 Linter.version; // => '9.0.0'
 ```
 
+### Linter#getSourceCode()
+
+This method is used to get the `SourceCode` object representing the parsed source code.
+
```

Comment:
This method was added in fc3bc8adde6cd428f7cbbce6505bd28df287d5e5. From the discussion in #948 it seems that it is meant to be a public API, and in that case it should be documented. The description is a bit unclear as it would return the source code of the last processed file, if any.